wikimedia/wikipedia-ios
# Scale Notifications Center cell's meta content image view as type size changes

**Phabricator:** Related to several tasks including https://phabricator.wikimedia.org/T297746

### Notes
This PR resolves a dynamic type issue with the meta content image view where the image would not appropriately scale to match the vertical height of the meta content text label.

### Test Steps

Confirm the meta content image and text at the bottom of Notifications Center cells scales as the accessibility dynamic type size changes.
